src/buildtools/buildtools.targets

```diff
@@ -43,7 +43,7 @@
           BeforeTargets="CoreCompile">
 
     <PropertyGroup>
-        <FsYaccPath Condition="'$(FsYaccPath)' == ''">$(ArtifactsDir)\Bootstrap\fsyacc\fsyacc.dll</FsYaccPath>
+        <FsYaccPath Condition="'$(FsYaccPath)' == ''">$(ArtifactsDir)\Bootstrap\fsyacc.dll</FsYaccPath>
```

Comment:
Why is this being removed? The different apps involved in the bootstrap tools should be published to different directories, because they may use incompatible dependencies.

Reply:
These tools are always going to build to the same target framework as FSC and FSI at on coreclr.

We could certainly make the bootstrap folder:
bootstrap\ {toolname{fsc|fsi|fslexx|fsyacc} .. \{netcoreapp2.1|net472}
or 
bootstrap\{netcoreapp2.1|net472}

I think it's 6 of 1 half a dozen of the other, either way they don't need publish, dotnet build should be good enough

Reply:
> These tools are always going to build to the same target framework as FSC and FSI at on coreclr.

The different tools can easily end up using different FSharp.Core versions and have various other dependencies.  

So we should use this: 

    bootstrap\ {toolname{fsc|fsi|fslexx|fsyacc} .. {netcoreapp2.1|net472}